python/paddle/fluid/layers/nn.py

```diff
@@ -5649,6 +5650,52 @@ def flatten(x, axis=1, name=None):
     return out
 
 
+def sequence_enumerate(input, win_size, pad_value, name=None):
+    """
+    Generate a new LoDTensor 
```

Comment:
Generate a new sequence for the input index sequence, which enumerates  all the sub-sequences with length `win_size` of the input. The enumerated sequence has the same 1st dimension with Variable `input`,  and the 2nd dimension is `win_size`, padded by `pad_value` if necessary in generation.

Reply:
Done